### PR TITLE
O3-780: Add bundle size monitoring to PR actions for all MF repos

### PR DESCRIPTION
**Pull request by FlorianRappl**
## Requirements

- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [x] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

We should be tracking bundle size changes. We want a tool that comments on every PR with the bundle size change caused by the PR.

## Screenshots

_None._

## Related Issue

See [O3-780](https://issues.openmrs.org/browse/O3-780).

## Other

_None._

### CODE REVIEW

**Comment by FlorianRappl** [comment]
The error above should be gone once the branch is checked in. `master` does not have the file currently. So I can only verify it when we merged in the change.

File: tools/size-generator.mjs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { generateFileSizeReport } from "@jsenv/file-size-impact";
+
+new URL("./", import.meta.url);
```
Comment (manuelroemer) [inline tools/size-generator.mjs:3]: Not sure, is this intended (maybe to throw an error for some weird URLs) or is this a leftover?

**Review by manuelroemer — APPROVED** [review]
Looks good, let's see this in action! 🚀

File: tools/size-generator.mjs (hunk 1 above)
Comment (FlorianRappl) [reply tools/size-generator.mjs]: Ha. You found my easter egg! Seriously, great catch : rocket:!

Comment (manuelroemer) [reply tools/size-generator.mjs]: The easter bunny came early this year 😆

**Pull request merged**